src/goto-programs/goto_function.h

```diff
@@ -17,13 +20,19 @@ Date: May 2018
 
 #include "goto_program.h"
 
+/// A goto function, consisting of function type, function body, and parameter
+/// identifiers.
 class goto_functiont
 {
 public:
   goto_programt body;
+
+  /// The type of the function, indicating the return type and parameter types
   code_typet type;
 
   typedef std::vector<irep_idt> parameter_identifierst;
+
+  /// The identifiers of the parameters of this function
```

Comment:
Yes, but see #1695.

Reply:
I've added a comment to #1695. I'll add a comment here to say that `parameter_identifiers` is currently always empty and that for now they should be retrieved from `code_typet`.